### DISCUSSION TITLE
Home favorite

### DISCUSSION THREAD

**Discussion by ramondevic** [devemux86/cruiser#428] [Q&A] [answered]
I think it would be interesting to have a favorite place by default that I can go to quickly, for example I have a list with my usual favorites and the first one is "home" but every time I want to go there I have to do: "my places > favorites > choose the group that interests me > home".
I wish I could do "I want to go home"

Maybe you could mark a "default favorite", just one and if another day you want to change the mark and put it in another one that from that moment on it will be "home" or any other name that is defined but that in all lists only and have a global favorite.

**Comment by MotoUKRider** [comment]
Sounds like an ideal feature to work into [my earlier suggestion for the Navigate button when empty.](https://github.com/devemux86/cruiser/discussions/64) :crossed_fingers:

**Comment by devemux86** [comment, marked as answer]
@ramondevic thanks for the suggestion.

Do you mean in the groups / favorites list or with another way?

Note: in favorites we can drag a group or favorite to the top of the list to find it quickly.

**Reply by ramondevic** [reply]
I mean being able to define a favorite that is the usual destination and with a single click start the route calculation and subsequent navigation, without having to navigate through the menu.

**Reply by devemux86** [reply]
Then we need a way to use a favorite location without the menus and using extra buttons.

**Reply by ramondevic** [reply]
Yes !!!
